lib/omniauth/azure_oauth2/version.rb

```diff
@@ -1,5 +1,5 @@
 module OmniAuth
   module AzureOauth2
-    VERSION = "0.0.12"
+    VERSION = "0.0.13"
```

Comment:
Why bump the version? In the Gemfile we'll be specifying this repo, right?

Reply:
Did it because that seemed to be [the practice](https://github.com/mode/omniauth-azure-oauth2/commits/master/)! We will be specifying the commit hash and repo, there is no need for this technically.

Reply:
Hmm, interesting. I guess it doesn't really matter.

Reply:
I have removed the version number change!

I am not sure if you have already observed this so I am putting this out here FYI: The current web app Gemfile does not refer to the latest master. It refers to a [commit id](https://github.com/mode/webapp/blob/6a548076cf254fd760eb0bdc36c7e56b5b828813/Gemfile#L34) which is [3 commits behind](https://github.com/mode/omniauth-azure-oauth2/compare/d98a3b8a74c3211cfd3268375330a8739c08b218...master) the latest master. These 3 commits don't have any code change (one adds code owners, one merges it, and the last bumps the version from .10 to .12.) When I merge this and take the hash from the latest master I will end up including those new changes.